app/services/stripe/invoice_handler.rb

```diff
@@ -0,0 +1,36 @@
+class InvoiceHandler
+  def initialize(event)
+    @invoice = event.data.object
+  end
+
+  def create_invoice!
+    begin
+    	Invoice.create!(
+        brand:              get_brand,
+        stripe_invoice_id:  @invoice.id,
+        amount:             @invoice.amount_due,
+        data:               @invoice,
+        paid_at:            Time.now.utc
```

Comment:
Noo, `created_at`  is set on `invoice.created`
And `paid_at` is set on `invoice.payment_succeeded`.